specs/deposits.md

```diff
@@ -349,13 +349,16 @@ the genesis file will be located in the `deployedBytecode` field of the build ar
 
 #### Ecotone L1Block upgrade
 
-The L1 Attributes Predeployed contract, `L1Block.sol`, is upgraded in the Ecotone upgrade.
+The L1 Attributes Predeployed contract, `L1Block.sol`, is upgraded as part of the Ecotone hardfork.
```

Comment:
```suggestion
The L1 Attributes Predeployed contract, `L1Block.sol`, is upgraded as part of the Ecotone upgrade.
```

In general we try to call it "upgrade", not "hardfork", since we build these improvements for consideration by the canonical chain, and would revisit and not fork away if these changes are not accepted.